### PR TITLE
Remove duplicate IN_Init call

### PR DESCRIPTION
**Pull request by slipher**

### CODE REVIEW

File: src/engine/qcommon/common.cpp [hunk 1]
```diff
@@ -576,10 +576,9 @@ Com_Crash_f
 A way to force a bus error for development reasons
 =================
 */
-static void NORETURN Com_Crash_f()
+static void Com_Crash_f()
 {
 	* ( volatile int * ) 0 = 0x12345678;
```
Comment (bmorel) [inline src/engine/qcommon/common.cpp:581]: maybe document what's this line is while you're at it?

**Review by bmorel — APPROVED** [review]
LGTM

File: src/engine/qcommon/common.cpp (hunk 1 above)
Comment (slipher) [reply src/engine/qcommon/common.cpp:581]: Crashing of course :)

Comment (bmorel) [reply src/engine/qcommon/common.cpp:581]: Ah, well, yes didn't noticed the *0, I was focused on the 0x12346.... silly me

**Pull request merged**